kafka/sink/README.md

```diff
@@ -0,0 +1,150 @@
+# Kafka Sink: Knative Sink for Apache Kafka 
+
+The Apache Kafka Event sink enables Knative Eventing integration with Apache
+Kafka. When a CloudEvent is received over HTTP by the Knative Sink it is
+sent to the topic in Apache Kafka broker.
+
+## Deploying Kafka Sink CRDs and controller
+
+TODO: coming soon
+
+## Deploying Kafka Sink to Kubernetes
```

Comment:
I wonder, can we show an example with a sinkbinding?

Reply:
Do you mean example that shows how to use kafkasink instead of event-display? https://knative.dev/docs/eventing/samples/sinkbinding/